docker/ubuntu/baseimage-20.04/install_luarocks.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+HOME=/home/ko
+
+cd $HOME || exit
+
+git clone https://github.com/torch/luajit-rocks.git
+pushd luajit-rocks && {
+    git checkout 2c7496b905f6f972673effda4884766433b7583b
+    cmake . -DWITH_LUAJIT21=ON -DCMAKE_INSTALL_PREFIX=${HOME}/local
+    make install
+} && popd || exit
+rm -rf luajit-rocks
```

Comment:
@Frenzie 

Can we update luajit & luarocks please :)

The system luarocks on 20.04 is newer  than this